packages/vit-s/src/hooks.js

```diff
@@ -199,7 +200,41 @@ export function useClosestVitessceContainerSize(ref) {
   return [width, height];
 }
 
-export function useExpressionValueGetter({ instanceObsIndex, matrixObsIndex, expressionData }) {
+export function useUint8ObsFeatureMatrix({ obsFeatureMatrix }) {
+  return useMemo(() => {
+    if (obsFeatureMatrix && obsFeatureMatrix.data) {
+      const dataExtent = extent(obsFeatureMatrix.data);
+      const [min, max] = dataExtent;
+      const ratio = 255 / (max - min);
+      const data = new Uint8Array(
+        obsFeatureMatrix.data.map(i => Math.floor((i - min) * ratio)),
+      );
+      return [{ data }, dataExtent];
+    }
+    return [null, null];
+  }, [obsFeatureMatrix]);
+}
+
+export function useUint8ExpressionData(expressionData) {
```

Comment:
Could these get doc strings?  And maybe also more informative names?  I think they more serve different purposes i.e., normalizing in different contexts, or no?